static/app/components/replays/header/replayMetaData.tsx

```diff
@@ -24,13 +24,13 @@ function ReplayMetaData({replayErrors, replayRecord}: Props) {
   const referrer = getRouteStringFromRoutes(routes);
   const eventView = EventView.fromLocation(location);
 
-  const domEventsTab = {
+  const breadcrumbTab = {
     ...location,
     query: {
       referrer,
       ...eventView.generateQueryStringObject(),
-      t_main: 'dom',
-      f_d_type: 'ui.slowClickDetected',
+      t_main: 'breadcrumbs',
+      f_b_type: 'rageOrDead',
```

Comment:
woo, looks like this already targets the new name per https://github.com/getsentry/sentry/pull/57942